personalize.module

```diff
@@ -1021,22 +1021,12 @@ function _personalize_agent_get_starttime_variable($agent_name) {
  *   - text to display
  */
 function _personalize_status_toggle_next($current_status) {
```

Comment:
In earlier attempts, this file was refactored to accommodate the story. Although later, changes in this file were no longer required for the story, I am keep the changes because it resolves a @todo item.